### PR TITLE
Bump cookbook version so berks will use a newer optoro_monit version.

### PR DESCRIPTION
**Pull request by jgillis01**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -19,3 +19,11 @@ suites:
...
     - recipe[apt]
     - recipe[optoro_memcached::default]
+  attributes:
+    monit:
+      poll_start_delay: 0
+      mailserver:
+        host: localhost
+        port: 25
+      mail_format:
+        from: monit@localhost
```
Comment (jgillis01) [inline .kitchen.yml]: Without a `from` field in the mail format monitrc configuration, the monit service will not (re)start, and the chef run will fail.

File: attributes/default.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 default['optoro_memcached']['log_dir'] = '/var/log/memcached'
 default['optoro_memcached']['log_path'] = "#{node['optoro_memcached']['log_dir']}/memcached.log"
 default['optoro_memcached']['mem_size_mb'] = '512'
+default['monit']['address'] = 'localhost'
```
Comment (zadunn) [inline attributes/default.rb]: Is this just the "from" base domain?  Or is it the complete address?  If it's the "from" can we make it the node[:fqdn] or something meaningful - Not a blocker.

File: files/default/newrelic_plugin_com.newrelic.plugins.memcached [hunk 3]
```diff
@@ -1,82 +0,0 @@
...
-    status
-    ;;
-  restart)
-    restart
-    ;;
-  *)
-    echo "Usage: ${0} {start|stop|restart|status}"
-    exit 1
-    ;;
-esac
```
Comment (zadunn) [inline files/default/newrelic_plugin_com.newrelic.plugins.memcached]: SOOO MUCH GLORIOUS READ

File: attributes/default.rb (hunk 2 above)
Comment (jgillis01) [inline attributes/default.rb]: Its the base domain, but it is populated via the original monit cookbook. So we have to override it anyway with our information.

**Pull request merged**